### ISSUE TITLE
GPIO pins on Adafruit Feather 2040

### ISSUE THREAD

**Issue by ehrhartd** [jfedor2/hid-remapper#190]
Would it be possible to assign keystrokes to the GPIO pins?  It would be nice to be able to assign keystrokes to the GPIOs for accessibility switches.

**Comment by jfedor2** [comment]
Yes, it's possible, you can set GPIO pins as inputs on mappings.

**Comment by ehrhartd** [comment]
That was a dumb question on my part. I guess I just never really ventured that far down into the inputs.  Sorry about that.  :-)

**Comment by goathens** [comment]
Sorry to necro, how many pins on the ada feather usbhost are exposed in the web interface? I'm constructing an arcade controller type device and want to have 12-16 buttons in my design, and use hid-remapper to allow me to write different controller profiles onto the feather.

**Comment by jfedor2** [comment]
@goathens I think there are 19 GPIO pins available for use on the Feather.

**Comment by dglaude** [comment]
I am also using HID-Remapper to turn GPIO from an arcade cabinet to USB control.
I have to cover
2 joystick => 8 GPIO
8 buttons => 8 GPIO
I decided to go with a Pico as I did not identify the need for USB input.
I feel that there are more GPIO exposed on the Pico than on the Feather (while both use the RP2040). But I don't know how many are supported in the code, at least I found (the 16 pin I need without problem).

Now I need to define a mapping.

**Comment by goathens** [comment]
I require the USB host, as I have another usb-based input device for the feather/hid-remapper to remap in combination with buttons on the GPIO pins.
